app/controllers/katello/api/v2/host_packages_controller.rb

```diff
@@ -20,10 +20,12 @@ class Api::V2::HostPackagesController < Api::V2::ApiController
 
     api :GET, "/hosts/:host_id/packages", N_("List packages installed on the host")
     param :host_id, :number, :required => true, :desc => N_("ID of the host")
+    param :packages_restrict_latest, :boolean, :desc => N_("Return only the latest version of each package")
```

Comment:
Is this parameter correctly named? It seems like less of a restriction but more of an "also include this extra data"

Reply:
I'm guessing this was to be consistent with other params; @parthaa and I had a discussion here: https://github.com/Katello/katello/pull/9742#discussion_r739527215

Reply:
But I don't think the name or description matches what the param actually does. I prefer accuracy over consistency!

Reply:
I agree!

Reply:
Pending meethig this morning: `include_latest_upgradable` - Also include the latest upgradable package version for each host package